Packs/SentinelOne/Integrations/SentinelOne-V2/SentinelOne-V2.yml

```diff
@@ -2631,6 +2654,149 @@ script:
     - contextPath: SentinelOne.Accounts.Name
       description: The account name.
       type: string
+  - arguments:
+    - name: account_ids
+      description: A comma-separated list of account IDs.
+      isArray: true
+    - name: application_catalog_id
+      description: Filter results by application catalog id. 
```

Comment:
```suggestion
      description: Filter results by application catalog ID. 
```

Reply:
It will not change because it is not related to the above contribution but to the commit from master.